user_guide/entity_management.rst

```diff
@@ -0,0 +1,255 @@
+   type, auditabilty and visibility in the grid.
+  
+6. Create at least one :ref:`field <user-guide-entity-management-create-fields>` for the entity.
+
+7. :ref:`Update <user-guide-entity-management-create-update>` the schema.
+
+
+.. _user-guide-entity-management-create-general-information:
+
+General Information
+^^^^^^^^^^^^^^^^^^^
+
+The following two fields are mandatory and **must** be defined:
+
+.. csv-table::
+  :header: "Field", "Description"
+  :widths: 10, 30
+
+  "**Name***","Name used to refer to the entity in the system.
+  The name shall be at least 5 characters long, may contain numbers and alphabetic symbols. The first symbol must be a 
+  letter.
+
+  The name field cannot be a `reserved sql word <http://msdn.microsoft.com/en-us/library/ms189822.aspx>`_"
+  "**Label***","The label will be used to refer to the entity in the UI. Label shall be at least 2 symbols long."
+
+.. caution::
+
+  **The Name value must be unique for every Custom entity created.** Fault to do so may cause partial update of 
+  the existing entity and failure of the schema update.
+  
+  Labels may duplicate, though from the UI perspective it will be impossible to distinguish
+  them, so it is better to keep them unique.
+    
+The following general entity details are optional and may be specified:
+
+.. csv-table::
+  :header: "Field", "Description"
+  :widths: 10, 30
+
+  "**Icon**","All the entity records will be displayed with the icon chosen."
+  "**Plural Label**","The label defined will be used to name the menu items and grids related to the entity."
+  "**Description**","Free text info to help you or other users understand the purpose and specifics of the created 
+  entity in future."  
+
+  
+.. _user-guide-entity-management-create-commun-collab:
+
+Communication & Collaboration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The section contains check-boxes for all the activities available and *Enable Notes* drop-down.
+
+.. image:: ./img/entity_management/new_entity_communication_collaboration.png
+
+- Check the activities that you want to enable for the entity.
+
+- Choose *Yes* in the drop-down and it will be possible to add free text *Notes* to the entity records.
+
+
+.. _user-guide-entity-management-create-attachments:
+
+Attachment Settings
+^^^^^^^^^^^^^^^^^^^
+
+You can enable attachments for the entity records and define relevant limitations.
+
+.. image:: ./img/entity_management/new_entity_attachment.png
+
+- Choose *Yes* in the drop-down to enable attachments.
+
+- Define the upper limit of an attachment size, if necessary. No larger attachments will be added.
+
+- Limit the list of supported MIME types if necessary. If no MIME type is defined, the list defined in 
+  the *System → Configuration → Upload Settings* will be used. 
+  
+  Define the list of MIME types supported in the field, add each new type on a new line. For example:
+  
+.. image:: ./img/entity_management/new_entity_general_MIME_types.png
```

Comment:
Thanks a whole lot. This is one of the things file generation doesn't help checking (as it is case insensitive). Had it corrected